fontdrasil/README.md

```diff
@@ -0,0 +1,9 @@
+# fontdrasil
+
+[![MIT/Apache 2.0](https://img.shields.io/badge/license-MIT%2FApache-blue.svg)](#license)
+
+fontdrasil: Connects all layers of fontc. A reference to Yggdrasil, the world tree.
```

Comment:
```suggestion
fontdrasil: Common types and functionality shared between all layers of fontc.
```

Reply:
Less Yggdrasil is clearly worse. I'll take your preamble but leave the Yggdrasil.